dashboard/lib/pd/foorm/survey_results.rb

```diff
@@ -0,0 +1,48 @@
+# Retrieves, parses and summarizes Foorm Survey results for consumption by APIs.
+module Pd::Foorm
+  class SurveyResults
+    extend Helper
+    include Constants
+
+    def self.get_summary_for_workshop(workshop_id)
+      return unless workshop_id
+
+      ws_data = Pd::Workshop.find(workshop_id)
```

Comment:
This is such an interesting function that it would be good to comment on each step of the process.